go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	cloud.google.com/go v0.56.0
 	cloud.google.com/go/spanner v1.5.1
 	cloud.google.com/go/storage v1.6.0
-	github.com/Azure/azure-storage-blob-go v0.11.0
+	github.com/Azure/azure-storage-blob-go v0.13.0
```

Comment:
I a bit surprised that this got updated.

Reply:
@calvn gosnowflake uses it, is this a blocker?

Reply:
Understood. Nope, if snowflake requires the upgrade then that's fine.